automation/check-patch.setup.sh

```diff
@@ -12,8 +12,8 @@ export PATH=${GOPATH}/bin:${GOROOT}/bin:${PATH}
 export GOBIN=${GOROOT}/bin/
 mkdir -p $GOBIN
 
-echo 'Install Go 1.19'
-export GIMME_GO_VERSION=1.19
+echo 'Install Go 1.20'
+export GIMME_GO_VERSION=1.20
```

Comment:
if you want a followup you can do as we are doing on CNAO
auto read the version from go.mod using a script
and then one less place to update
(maybe same trick can be done with ARG for dockerfile)

Reply:
That would be nice to have.